axelor-stock/src/main/java/com/axelor/apps/stock/web/StockMoveLineController.java

```diff
@@ -262,10 +262,7 @@ public void displayAvailableTrackingNumber(ActionRequest request, ActionResponse
       StockLocationLine detailStockLocationLine =
           stockLocationLineService.getDetailLocationLine(
               stockMove.getFromStockLocation(), stockMoveLine.getProduct(), trackingNumber);
-      BigDecimal availableQty =
-          detailStockLocationLine != null
```

Comment:
Please keep this check. I agree that we should always find a detailStockLocationLine here but if there is an issue in the future it's better to avoid a NPE.